src/common/QLogging.cpp

```diff
@@ -1,9 +1,9 @@
 #include "common/QLogging.hpp"
 
-#ifdef DEBUG_OFF
-static constexpr QtMsgType logThreshold = QtWarningMsg;
-#else
+#ifndef NDEBUG
```

Comment:
Nitpick: if you keep this as `ifdef NDEBUG` this turns into a 1-line diff

Reply:
Done